optimism/src/mips/keccak/mod.rs

```diff
@@ -0,0 +1,67 @@
+pub mod column;
```

Comment:
On further reflection, shouldn't this be a separate module in `optimism/src` instead of living in MIPS?

Reply:
Sounds good, will move it outside